content/docs/alerting/configuration.md

```diff
@@ -405,8 +405,12 @@ api_key: <secret>
 # The host to send OpsGenie API requests to.
 [ api_host: <string> | default = global.opsgenie_api_host ]
 
+# Alert text limited to 130 characters
```

Comment:
There's no truncation done in the code if that's an actual API limit.

Comments should end in a full stop.

Reply:
@brian-brazil Got this text from the api documentation itself, where they say the limit exists. Should I remove it?
https://docs.opsgenie.com/docs/alert-api-deprecated

Reply:
It's more that we should implement it on the alertmanager side, as we do for some other limited fields.

Reply:
@brian-brazil Gotcha! Gonna create an issue there and work on it.